src/test/php/PHPMD/Rule/Controversial/CamelCasePropertyNameTest.php

```diff
@@ -42,15 +42,15 @@ public function testRuleDoesNotApplyForValidPropertyName()
     }
 
     /**
-     * Tests that the rule does NOT apply for an property name
+     * Tests that the rule does apply for an property name
```

Comment:
```suggestion
     * Tests that the rule does apply for a property name
```
While modifying this line, we could take the opportunity to fix this typo.

Reply:
Fixed.